extensions/all-extensions/pom.xml

```diff
@@ -29,6 +29,19 @@
             <groupId>org.mule.modules</groupId>
             <artifactId>mule-module-validation</artifactId>
             <version>${project.version}</version>
+            <type>zip</type>
+            <!--TODO(gfernandes): https://issues.apache.org/jira/browse/MASSEMBLY-799
+                wildcard exclusions were supported in maven-core https://issues.apache.org/jira/browse/MNG-3832
+                but still assembly it having not handling this correctly
+                so the transitive dependencies of this artifact will end up in lib/opt (and we don't want to explicity
```

Comment:
I would split this message in two: one saying that we must explicitly exclude any dependency included in the zip files, and another comment for the TODO in order to remove that limitation once the issue is fixed